.gitignore

```diff
@@ -110,6 +110,32 @@
 /windows/src/engine/keyman32/*.obj
 /windows/src/engine/keyman32/*.cod
 /windows/src/engine/keyman32/*.pdb
+/windows/src/engine/keyman32/exports.h
```

Comment:
Can we put all the new windows/src gitignore entries into the windows/src/.gitignore? Trying to keep them out of the top level.

Also, the etw-related files should only be in keyman32, as that's the only dll that should contain the etw resources.